metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -229,6 +258,29 @@ def create(cls, config_dict: dict, ctx: PipelineContext) -> Source:
         config = SupersetConfig.parse_obj(config_dict)
         return cls(ctx, config)
 
+    @lru_cache(maxsize=None)
```

Comment:
tiny nit - if the callers to this method have lru_cache, do we also need an lru_cache on this method?

Reply:
I guess not, don't think we're caching anything truly meaningful here. Will look to remove.
I actually had a question on caching. Will this cache persist over different ingestion runs or just the current one?

Reply:
this cache will not persist across runs